src/test/java/edu/ucla/library/bucketeer/handlers/LoadImageHandlerTest.java

```diff
@@ -195,4 +195,24 @@ public void confirmLoadImageHandlerFailsWithMissingParam(final TestContext aCont
         });
     }
 
+    /**
+     * Confirm that LoadImageHander fails if we provide and invalid filePath
+     *
+     * @param aContext A testing context
+     */
+    @Test
+    @SuppressWarnings("deprecation")
+    public void confirmLoadImageHandlerFailsWithInvalideFilepath(final TestContext aContext) {
+        final Async async = aContext.async();
+        final int port = aContext.get(Config.HTTP_PORT);
+
+        // Testing the main loadImage path defined in our OpenAPI YAML file returns an error response when given
+        // incomplete data
+        myVertx.createHttpClient().getNow(port, Constants.UNSPECIFIED_HOST, "/12345/this-file-does-not-exist.tiff",
+            response -> {
+                aContext.assertNotEquals(response.statusCode(), HTTP.OK);
+                async.complete();
```

Comment:
Not a big deal, but I've started wrapping this in an `if (!async.isCompleted()) { async.complete(); }` lately to avoid the warning message you get if the test fails (in this case you get a HTTP.OK -- which completes test with a failure) and then `async.complete()` is called after the test has already failed.

Reply:
fix now or later?

Reply:
Up to you, I approved as is.

Reply:
later: I've opened a ticket for the refactor: IIIF-492